src/main/java/space/obminyashka/items_exchange/controller/request/ChangePasswordRequest.java

```diff
@@ -0,0 +1,25 @@
+package space.obminyashka.items_exchange.controller.request;
+
+import io.swagger.v3.oas.annotations.Parameter;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import space.obminyashka.items_exchange.util.PatternHandler;
+
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+
+import static space.obminyashka.items_exchange.util.ResponseMessagesHandler.ValidationMessage.*;
```

Comment:
A blank line after the import is required

Reply:
Added